examples/src/main/python/ml/binarizer_example.py

```diff
@@ -17,9 +17,10 @@
 
 from __future__ import print_function
 
-from pyspark.sql import SparkSession
 # $example on$
 from pyspark.ml.feature import Binarizer
+from pyspark.sql import SparkSession
```

Comment:
Some of the examples files are used in generating the website documentation, and the "example on" and "example off" tags are used to determine which parts get pulled in to the website (in this case this is done since we don't want to have the same boiler plate imports for each example - rather showing the ones specific to that). You can take a look at `./docs/ml-features.md` which includes this file to see how its used in markdown and the generated website documentation at http://spark.apache.org/docs/latest/ml-features.html#binarizer .

The instructions for building the docs locally are located at `./docs/README.md` - let me know if you need any help with that - the documentation build is sometimes a bit overlooked since many of the developers don't build it manually often.